### PR TITLE
add what is needed by kinsol

### PR DESCRIPTION
**Pull request by asartori86**
to be merged after https://github.com/mathLab/deal2lkit/pull/297

### CODE REVIEW

**Comment by asartori86** [comment]
This is ready for review

File: include/interfaces/navier_stokes.h [hunk 1]
```diff
@@ -651,7 +651,12 @@ NavierStokes<dim,spacedim,LAC>::compute_system_operators(
   LinearOperator<LATrilinos::VectorType> &prec_op,
   LinearOperator<LATrilinos::VectorType> &prec_op_finer) const
 {
-  auto alpha = 0;
+  auto alpha = this->get_alpha();
+  if (alpha == 0.0)
+    {
+      std::cout << "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzero" << std::endl;
```
Comment (ESeNonFossiIo) [inline include/interfaces/navier_stokes.h]: magari lo `std::cout` lo eviterei... :D
Puoi tornare alla versione precedente di questo file che così pusho la mia una volta che Kinsol va?

File: source/pidomus.cc [hunk 2]
```diff
@@ -871,7 +896,7 @@ shared_ptr<typename LAC::VectorType>
 piDoMUS<dim, spacedim, LAC>::create_new_vector() const
 {
   shared_ptr<typename LAC::VectorType> ret = SP(new typename LAC::VectorType(solution));
-  *ret *= 0;
+  //   *ret *= 0;
```
Comment (ESeNonFossiIo) [inline source/pidomus.cc:899]: ?

File: include/pidomus.h [hunk 3]
```diff
@@ -242,6 +252,7 @@ class piDoMUS : public ParameterAcceptor, public SundialsInterface<typename LAC:
   LinearOperator<typename LAC::VectorType> jacobian_op;
 
 
+
```
Comment (nicola-giuliani) [inline include/pidomus.h]: ...gia' sento rumore di rotelle... ;)

**Comment by ESeNonFossiIo** [comment]
Per me, se fai il revert di `navier_stokes.h`, va bene

File: source/pidomus.cc (hunk 2 above)
Comment (asartori86) [inline source/pidomus.cc:899]: `create_new_vector()` is called from imex/ida/kinsol. It is supposed to overwrite this vector, so there **should** be no difference. I comment and uncomment it to double check that there is no difference between the two cases. Tell me what do you prefer

**Comment by asartori86** [comment]
@ESeNonFossiIo , I've just left `auto alpha = this->get_alpha();` such that tests pass.
@nicola-giuliani I've fixed your comment ;)

**Pull request merged**